src/test/regress/citus_tests/config.py

```diff
@@ -169,6 +172,17 @@ def __init__(self, arguments):
             # Alter Table statement cannot be run from an arbitrary node so this test will fail
             "arbitrary_configs_alter_table_add_constraint_without_name_create", "arbitrary_configs_alter_table_add_constraint_without_name"]
 
+class CitusGrantedPermissionsClusterConfig(CitusDefaultClusterConfig):
+    def __init__(self, arguments):
+        super().__init__(arguments)
+        self.create_user = SUPER_USER_NAME
```

Comment:
At first, I was confused that we have `create_user` and `user`. We also have `user` and `role` used in this file.

Maybe we make them more explicit by saying: `owner_role` and `session_role` or such?

Reply:
also, maybe explicitly add `self. session_role  = REGULAR_USER_NAME` so that it is much easier to follow (instead of tracing back the base file)